src/types/d2-ui-components.d.ts

```diff
@@ -0,0 +1,44 @@
+import { Moment } from "moment";
+
+export interface TableList {
+    objects: any[];
```

Comment:
Yeah, indeed. But since we are using the same type files in both the webapp and here, we should change it first there and then keep this copy in sync.